packages/cms-lib/api/fileManager.js

```diff
@@ -31,21 +31,31 @@ async function getStat(portalId, src) {
   });
 }
 
-async function getFilesByPath(portalId, src) {
+async function getFiles(portalId, folderId, { offset, archived }) {
   return http.get(portalId, {
-    uri: `${FILE_MANAGER_API_PATH}/files/path/${src}`,
+    uri: `${FILE_MANAGER_API_PATH}/files/`,
+    qs: {
+      hidden: 0,
+      offset: offset,
+      folder_id: folderId || 'None',
+      ...(archived === 0 && { archived }),
+    },
   });
 }
 
-async function getFoldersByPath(portalId, src) {
+async function getFolders(portalId, folderId) {
   return http.get(portalId, {
-    uri: `${FILE_MANAGER_API_PATH}/folders/path/${src}`,
+    uri: `${FILE_MANAGER_API_PATH}/folders/`,
+    qs: {
+      hidden: 0,
+      parent_folder_id: folderId || 'None',
+    },
   });
 }
 
 module.exports = {
   uploadFile,
   getStat,
-  getFilesByPath,
-  getFoldersByPath,
+  getFiles,
+  getFolders,
```

Comment:
FWIW, I tend to favor `fetch*` for GET request api helpers to help communicate their async nature when compared to typical "getters" that getting data in a synchronous fashion from somewhere.